velox/connectors/hive/storage_adapters/gcs/examples/GCSFileSystemExampleWrite.cpp

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
Can we add this example in a separate PR? It does not relate to the new configs added here.

Reply:
I wrote that example because it clearly shows the effect of leaving the default retry (15 minutes) and not authenticating, but you are right, I will provide it in another PR.